### PR TITLE
fix: fix namespace

### PR DESCRIPTION
**Pull request by pazolka**
```
NameError (undefined local variable or method `model_class' for #<WcmcComponents::TableController:0x00007fce7abf2c28>):
wcmc_components (0.4.0) app/controllers/wcmc_components/table_controller.rb:4:in `list'
```

changing `ApplicationController` -> `WcmcComponents::ApplicationController` fixes this issue

### CODE REVIEW

File: wcmc_components/app/controllers/wcmc_components/table_controller.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 module WcmcComponents
-  class TableController < ApplicationController
-    def index
-      @results = model_class.paginate_for_table(query_params_with_symbol_keys)
 
+  class TableController < WcmcComponents::ApplicationController
+    def list
```
Comment (WillKocur) [inline wcmc_components/app/controllers/wcmc_components/table_controller.rb:4]: I think you might have worked off an older version at some point - the action name here should be index (it used to be called list). 

A tiinnyyy thing, there's a blank line on line 2 which we don't usually have. Not a big deal but nice to be consistent :)

**Review by WillKocur — CHANGES_REQUESTED** [review]
Works nicely - glad you found this issue and got it sorted!

Just the one comment added to the code where you've used the old action name. Once that's changed, it'll be good to merge 👍

File: wcmc_components/app/controllers/wcmc_components/table_controller.rb (hunk 1 above)
Comment (pazolka) [reply wcmc_components/app/controllers/wcmc_components/table_controller.rb]: Oh yes! It must have happened when resolving the merge conflict.. Fixed now :)

Comment (WillKocur) [reply wcmc_components/app/controllers/wcmc_components/table_controller.rb]: Ah yeah, I imagined as much. Cheers!

**Pull request merged**